public/app/features/migrate-to-cloud/onprem/MigrationInfo.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactNode } from 'react';
+
+import { Stack, Text } from '@grafana/ui';
+
+export function MigrationInfo({ title, value }: { title: NonNullable<ReactNode>; value: NonNullable<ReactNode> }) {
+  return (
+    <Stack direction="column">
+      <Text variant="bodySmall" color="secondary">
+        {title}
+      </Text>
+      <Text variant="h4" element="span">
```

Comment:
minor nit: isn't `span` the default anyway?

Reply:
good shout - I couldn't remember where we landed on with "does `varient=h4` render a H4 by default?